warehouse/upload_test.go

```diff
@@ -160,7 +160,7 @@ var _ = Describe("Upload", Ordered, func() {
 	},
 		Entry(nil, []error{}, true),
 		Entry(nil, []error{&TableSkipError{}}, true),
-		Entry(nil, []error{errors.New("")}, false),
+		Entry(nil, []error{errors.New("some-error")}, false),
```

Comment:
How this test can pass regardless of the value of the error?

Reply:
`areAllTableSkipErrors` check if the error is of a specific type `TableSkipError`

Reply:
value doesn't matter.

Reply:
https://deepsource.io/gh/rudderlabs/rudder-server/run/2284a648-4dd4-41a9-8dc3-c3d87c9e1fff/go/GO-W1024?listindex=0